gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthGrantFinder.js

```diff
@@ -43,20 +43,30 @@ var fluid = fluid || require("infusion");
     // and the client credentials grant to find the match. The different data structure
     // can be returned based on the grant type.
     gpii.oauth2.authGrantFinder.getGrantForAccessToken = function (authorizationService, accessToken) {
```

Comment:
I think that it might be better to change the comment for `gpii.oauth2.authGrantFinder.getGrantForAccessToken` to discuss client types rather than grant types. What do you think?